src/rest-server/deploy/rest-server.yaml.template

```diff
@@ -93,6 +93,10 @@ spec:
           value: "{{ cluster_cfg['rest-server']['rate-limit-list-job-per-min'] }}"
         - name: RATE_LIMIT_SUBMIT_JOB_PER_HOUR
           value: "{{ cluster_cfg['rest-server']['rate-limit-submit-job-per-hour'] }}"
+        - name: SCHEDULE_PORT_START
```

Comment:
Doc and comment the impact after change this config.
Such as will it impact existing running jobs, and how

Reply:
Sure, the rest-server config doc seems out of date. Will sync with @abuccts about how to update it.
And it will not impact running job, the backward compatibility code is added: https://github.com/microsoft/pai/blob/8e57e41e3748d01bed0e8eb76d5572868edc3e39/src/rest-server/src/models/v2/job/k8s.js#L200-L203

Reply:
Sure, will do it